apps/next_umd/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "rambler",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "run-p start-server build-watch",
```

Comment:
BINGMAP does not work.
'''export BINGMAP_ACCESS_TOKEN=<bingmap_access_token>''' may be ignored.